source/apiVolontaria/apiVolontaria/serializers.py

```diff
@@ -58,16 +58,6 @@ def validate(self, attrs):
                 password=password
             )
 
-            if user:
-                if not user.is_active:
-                    # User is inactive
-                    msg = "This user is inactive"
-                    raise exceptions.ValidationError(msg)
-            else:
-                # Bad password
-                msg = "Bad password"
-                raise exceptions.ValidationError(msg)
-
```

Comment:
Ok for delete the `if not user.is_active:` but i don't understand why you delete `if user:` and `else:` condition.
If we don't find a user with `authenticate()` we need to send a message no ?

Reply:
You're right. In fact, if we don't make the check ourselves, the API returns 
```json
{
  error: "Could not authenticate user"
}
```
when you submit a bad password. This is a builtin error message. I'll see if I can make it more verbose.